command/v7/revision_command.go

```diff
@@ -14,7 +19,118 @@ type RevisionCommand struct {
 }
 
 func (cmd RevisionCommand) Execute(_ []string) error {
-	cmd.UI.DisplayWarning(command.ExperimentalWarning)
+	err := cmd.SharedActor.CheckTarget(true, true)
+	if err != nil {
+		return err
+	}
+
+	user, err := cmd.Config.CurrentUser()
+	if err != nil {
+		return err
+	}
+
+	appName := cmd.RequiredArgs.AppName
+	cmd.UI.DisplayTextWithFlavor("Showing revision {{.Version}} for app {{.AppName}} in org {{.OrgName}} / space {{.SpaceName}} as {{.Username}}...", map[string]interface{}{
+		"AppName":   appName,
+		"OrgName":   cmd.Config.TargetedOrganization().Name,
+		"SpaceName": cmd.Config.TargetedSpace().Name,
+		"Username":  user.Name,
+		"Version":   cmd.Version.Value,
+	})
+	cmd.UI.DisplayNewline()
+
+	app, warnings, err := cmd.Actor.GetApplicationByNameAndSpace(appName, cmd.Config.TargetedSpace().GUID)
+	cmd.UI.DisplayWarnings(warnings)
+	if err != nil {
+		return err
+	}
+
+	deployedRevisions, warnings, err := cmd.Actor.GetApplicationRevisionsDeployed(app.GUID)
+	cmd.UI.DisplayWarnings(warnings)
+	if err != nil {
+		return err
+	}
+
+	revision, warnings, err := cmd.Actor.GetRevisionByApplicationAndVersion(
+		app.GUID,
+		cmd.Version.Value,
+	)
+	cmd.UI.DisplayWarnings(warnings)
```

Comment:
*Question:* If we display the warnings in this line, are we not going to repeat them in lines 69 and 81 as well?